spring-cloud-azure-samples/spring-cloud-azure-config-sample/pom.xml

```diff
@@ -14,7 +14,7 @@
     <dependencies>
         <dependency>
             <groupId>com.microsoft.azure</groupId>
-            <artifactId>spring-cloud-azure-config</artifactId>
+            <artifactId>spring-cloud-starter-azure-config</artifactId>
```

Comment:
With this, we should not need `spring-boot-starter`

Reply:
Updated.